### DISCUSSION TITLE
Increasing the number of mesh

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#246] [General]
Greetings,

How can I increase the number of mesh size in peridigm/examples/tensile_test.

Please guide me

**Comment by djlittl** [comment]
Do you have access to Cubit or Trelis?

The mesh for the tensile_test example was created using Cubit, which is not freely available.  There is another version, called Trelis, that is geared toward commercial and academic users.  I have never used it, but it sounds promising.

https://coreform.com/products/cubit-trelis/

**Reply by Nikhil23121998** [reply]
Thank you very much.

**Reply by Nikhil23121998** [reply]
Is it possible to give fiber properties to tensile_test example with the help of cubit?

**Comment by djlittl** [comment]
The commands for generating the mesh in Cubit are stored in the file tensile_test.jou, which is just a text file.  You can reproduce the mesh file (tensile_test.g) by copy-and-pasting all the contents of tensile_test.jou into the command-line window in Cubit.

I think the only required change is that you have to change the path for the output file to a path that actually exists on your machine.  This is at the bottom of tensile_test.jou.

Each of the commands in tensile_test.jou can also be executed in Cubit using the Cubit GUI, but this can be tedious.  The GUI is good for experimenting with different commands as you work on the model, but once you have a mesh you like, it can be a good idea to store the commands in a text file, which is what was done to create tensile_test.jou.

If you want to change the geometry or the mesh characteristics, you can alter the commands in tensilte_test.jou and re-run it in Cubit, or you could play around with the various Cubit commands using the GUI.

Dave

From: Nikhil23121998 ***@***.***>
Reply-To: peridigm/peridigm ***@***.***>
Date: Tuesday, August 9, 2022 at 3:14 AM
To: peridigm/peridigm ***@***.***>
Cc: "Littlewood, David John" ***@***.***>, Comment ***@***.***>
Subject: [EXTERNAL] Re: [peridigm/peridigm] Increasing the number of mesh (Discussion #246)


Thank you very much, can I open genesis file in cubit or I have to change its format first.

actually I am new in this field so if my questions are bit silly, please pardon me.

**Reply by Nikhil23121998** [reply]
Thank you
